src/start-app/components/PersonResolver/usePersonResolver.tsx

```diff
@@ -8,49 +8,40 @@ const createPersonClient = (client: IHttpClient) => {
   // TODO - good cache amount? 3min?
   const expire = 3 * 60 * 1000;
   const queryDetails = new Query({
-    expire,
-    queueOperator: 'merge',
-    key: (azureId) => azureId,
-    client: {
-      fn: async (azureId: string) => {
-        const user = await client.json<PersonDetails>(`/persons/${azureId}?api-version=4.0&$expand=positions,manager`);
-
-        try {
-          const image = await client.json<string>(`/persons/${azureId}/photo?api-version=1.0`);
```

Comment:
Will the image not be included now?

Reply:
No, There is no pictureSrc on the latest version of PersonDetails interface:
```ts
export type PersonInfo = {
  azureId: string;
  name?: string;
  jobTitle?: string;
  department?: string;
  mail?: string;
  upn?: string;
  mobilePhone?: string;
  accountType?: PersonAccountType[keyof PersonAccountType];
  officeLocation?: string;
  managerAzureUniqueId?: string;
};

export type PersonDetails = PersonInfo & {
  positions?: Position[];
  manager?: Manager;
};
```

Reply:
and this is a hotfix to resolve non functioning apps in the portal for mismatching fusion-component and fusion-api, if we need improvements to personResolver we need another release. i just made changes to it to suit the latest fusion-react-person component.

Reply:
any thought on this?

Reply:
Using `<PeropleResolverProvider>` and this will not be used.